### DISCUSSION TITLE
Small mobile touchscreen

### DISCUSSION THREAD

**Discussion by Deezelchonk** [ppy/osu#19070] [Ideas] [answered]
So I have been a tester for a while on mobile then I found this problem with mobile mania mode, the touchscreen isn’t the same as the other rhythm games (one’s with mobile support) because the difference is that osu!lazer’s mobile touchscreen, you have to tap or touch on the notes/sliders manually or directly (or tapping only the black lines to work)
My idea would be making the touchscreen have like 4 quarters that would cover most of the screen (the menu button should be somewhere else so it wouldnt interrupt the gameplay that easy) if it was 4keys (more if more keys) so it would be easier to hit the notes (if anyone still finds this confusing, I will try to explain the context of this)

**Comment by peppy** [comment, marked as answer]
There are at least 10 and probably more like 100 discussions open on this topic. Please use the search field.

**Reply by Deezelchonk** [reply]
I searched for those but it's just that most of them didn't relate much to very small touchscreen (required to manually press on the black lines to hit the notes but is a very big problem for a much more comfortable gameplay since your fingers will barely or not have any space for it to be easier to hit the notes) for mania.

**Reply by fyyrraaa** [reply]
have a look at #16783
